int32 MaxContacts
float64 JointNum
bool   FixedBase
float64 TotalArmLen
float64 LinkRad
float64 FSThick
float64 FSSize
float64 BaseLenX
float64 BaseLenY
float64 BaseLenZ
float64 GBaseLenX
float64 GBaseLenY
float64 GBaseLenZ
float64 GripLenY
float64 GripLenZ
int32 ObjectMode
float64 Box1PosX
float64 Box1PosY
float64 Box1SizeX
float64 Box1SizeY
float64 Box1SizeZ
float64 Box1Density1
float64 Box1Density2
float64 TimeStep
float64 Gravity
bool EnableKeyEvent
float64 HingeFMax
float64 SliderFMax
int32 ControlMode
